devel/pijul-devel/Portfile

```diff
@@ -0,0 +1,402 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cargo 1.0
+PortGroup           openssl 1.0
+
+set pname           pijul
+name                ${pname}-devel
+version             1.0.0-beta.6
+categories          devel
+license             GPL-2+
+maintainers         nomaintainer
+description         ${pname} is a free and open source distributed version control system
+long_description    ${description}. Its distinctive feature is to be based on a sound theory, \
+                    which makes it easier to learn and use, and really distributed.
+homepage            https://${pname}.org/
```

Comment:
Nitpick: There is no need to replace `pijul` with `${pname}` in homepage. It's a complete URL, not a common pattern where `${pname}` can be replaced with other names.